### ISSUE TITLE
Exception raised when publishing on a new machine with no authenticated user

### ISSUE THREAD

**Issue by kipcole9** [hexpm/hex#516]
I've moved to a new machine and on the first `mix hex.publish` from that machine an exception was raised.  A subsequent `hex.publish` was successful. Hex 17.3, Elixir 1.7.0-dev:
```elixir
iex>mix hex.publish
...
No authenticated user found. Do you want to authenticate now? [Yn] 
Username: kipcole9
Account password: 
Generating API key...
You have authenticated on Hex using your account password. However, Hex requires you to have a local password that applies only to this machine for security purposes. Please enter it.
Local password: 
Local password (confirm): 
Generating organization keys...
** (MatchError) no match of right hand side value: :error
    (hex) lib/hex/crypto/encryption.ex:30: Hex.Crypto.Encryption.decrypt/2
    (hex) lib/mix/tasks/hex.ex:161: Mix.Tasks.Hex.auth_info/0
    (hex) lib/mix/tasks/hex.publish.ex:149: Mix.Tasks.Hex.Publish.create/3
    (mix) lib/mix/task.ex:314: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:80: Mix.CLI.run_task/2
```

**Comment by supersimple** [comment]
Thanks @kipcole9.
@ericmj @wojtekmach I was able to reproduce the error. The issue is that in this case `Hex.Crypto.Encryption.decrypt/2` is receiving cipher_text as a 3 member tuple where it expects a binary `def decrypt({tag, cipher_text}, opts) do`. I will get a PR up shortly with a fix.

**Comment by supersimple** [comment]
PR https://github.com/hexpm/hex/pull/519 opened.

**Comment by wojtekmach** [comment]
fixed in #519
